charts/backbeat/templates/lifecycle/conductor_deployment.yaml

```diff
@@ -48,6 +48,14 @@ spec:
               value: "{{ .Values.lifecycle.rules.noncurrentVersionExpiration.enabled }}"
             - name: EXTENSIONS_LIFECYCLE_RULES_ABORT_INCOMPLETE_MPU_ENABLED
               value: "{{ .Values.lifecycle.rules.abortIncompleteMPU.enabled }}"
+          livenessProbe:
+            httpGet:
+              path: /_/health/liveness
+              port: 4042
```

Comment:
Use named ports instead of numeric ports (this applies to all occurrences...)

Reply:
Instead of using named ports, which would require that they be exposed, move the config into values.yml